packages/radio-player/src/assets/img/right.ts

```diff
@@ -0,0 +1,17 @@
+import { html } from 'lit-html';
```

Comment:
question: what is the difference between `right.svg` and `right.ts`?

Reply:
I removed the `svg` files... we were only using the `ts` files